docs/sources/reference/commandline/cli.md

```diff
@@ -1160,6 +1164,12 @@ You'll need two shells for this example.
     2014-05-10T17:42:14.999999999Z07:00 7805c1d35632: (from redis:2.8) die
     2014-09-03T15:49:29.999999999Z07:00 7805c1d35632: (from redis:2.8) stop
 
+    $ docker events --since '3m'
```

Comment:
There are no examples for --until at all already as far as I can see? So I just preserved the status quo about that in this PR. I'm not sure if there's value in adding.